bob_the_builder/src/pkg_build.rs

```diff
@@ -0,0 +1,109 @@
+use std::{collections::HashMap, fs, path::Path, process::Command};
+
+use serde::Deserialize;
+
+use crate::cargo_toml::package::{Build, BuildName, BuildSettings};
+
+#[derive(Deserialize, Debug, Eq, PartialEq)]
+pub struct ParsedPackage {
+    pub name: String,
+    pub builds: Vec<Build>,
+    pub default_build: bool,
+}
+
+impl ParsedPackage {
+    /// Build a contract with all the requested builds defined in `[package.metadata.optimizer]`
+    pub fn build(self, path: &Path) {
+        // Keep track of the unique builds to prevent re-compiling the same contract
+        // TODO: Reconsider this caching layer (or make it optional)
```

Comment:
I think for now we should remove this. Primarily for simplicity but also because you can organize your build config in a way that you don't need it. E.g. you don't need one config per network but with the approach we designed can do something like "with-tokenfactory" or "cosmwasm2" and use those builds for multiple chains. Having the same features configured for multiple builds seems to be a niche case to me that can easily also be handled with build names like "juno_or_chihuaua_or_neutron".